cmd/snap/cmd_wait.go

```diff
@@ -75,10 +78,46 @@ func trueish(vi interface{}) bool {
 		if v > 0 {
 			return true
 		}
+	case int8:
+		if v > 0 {
```

Comment:
Nitpick: I notice that this is not related to the PR as the conditions was there before but I would consider -1 "truish" as well and would rewrite the test to `v != 0`

Reply:
ah, I missed this

Reply:
Indeed, sorry for that.